addon/components/utils/address-form.ts

```diff
@@ -83,4 +125,50 @@ export default class extends Component<UtilsAddressFormArgs> {
       return isEmpty(get(this.args.address, addressAttr)) || shortAddress || invalidCountryCode || invalidZipcode;
     });
   }
+
+  private fillInAddress(place: any): void {
+    let address1: string = '';
+    let zipcode: string = '';
+    let city: string = '';
+
+    for (const component of place.address_components.reverse()) {
+      const componentType = component.types[0];
+
+      switch (componentType) {
+        case 'street_number': {
```

Comment:
how about something like : 
```
const mapper = {
  'street_number': () => (address1 = `${component.long_name} ${address1}`),
  'route': () => (address1 += component.long_name),
  'postal_code': () => `${component.long_name}${zipcode}`,
  ...
};
```
Note, using parenthesis instead of `{` will probably gain more space

Reply:
might be shorter but i find it harder to read :/